amgut/lib/data_access/sql_connection.py

```diff
@@ -53,6 +53,7 @@ def __init__(self):
         self._connection = None
         self._post_commit_funcs = []
         self._post_rollback_funcs = []
+        self.auto_rollback = False
```

Comment:
what about a reset of the test database? It will be consistent with the other projects.